auto_nag/scripts/workflow/p3_p4_p5.py

```diff
@@ -28,13 +28,19 @@ def has_product_component(self):
     def columns(self):
         return ['component', 'id', 'summary']
 
+    def handle_bug(self, bug, data):
```

Comment:
maybe move that stuff in utils?

Reply:
handle_bug is defined in bzcleaner class and so we need to overwrite it to filter the bugs